war/src/main/webapp/scripts/msie.js

```diff
@@ -34,16 +34,3 @@ function assertDocumentModeOK() {
 }
 
 assertDocumentModeOK();
-
-// IE 8 and before tweaks....
-if (YAHOO.env.ua.ie && YAHOO.env.ua.ie < 9) {
-    function loadRespondJS() {
-        var head = document.getElementsByTagName('head')[0];
-        var script = document.createElement('script');
-
-        script.setAttribute('src', head.getAttribute('resURL') + '/scripts/respond.js');
```

Comment:
shouldn't respond.js also be removed?

Reply:
Likely, a quick search shows that this is the only place it was used

Reply:
Yes you are right. Should I remove it within this PR or create a new one?

Reply:
in this PR, it's a polyfill for this functionality

Reply:
done.